.cirrus.yml

```diff
@@ -118,11 +118,11 @@ mac_task:
     populate_script:
       - export MACOSX_DEPLOYMENT_TARGET=10.10
       - python -m pip install --retries 3 --upgrade pip
-      - python -m pip install --retries 3 -r requirements/test.txt
+      - python -m pip install --retries 3 -r requirements/test.txt --no-cache-dir
 
   install_script:
     - python -m pip install --retries 3 --upgrade pip
-    - python -m pip install --retries 3 -r requirements/test.txt
+    - python -m pip install --retries 3 -r requirements/test.txt --no-cache-dir
```

Comment:
why use the `--no-cache-dir` option instead of just getting rid of the `pip_cache` step in the first place?  With this change, we're basically saying "cache it, but don't ever use the cache!" right? :)

also... is this a one-time and then switch-it-back sort of thing?  Or have you learned that we will no longer be able to take advantage of caching at all to speed up builds?

Reply:
I was hoping this would clear the cache so we don't need to merge anything, but I am not very confident